stores/directory/local.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod, abstractstaticmethod
+from pathlib import Path
+from typing import List
+
+from stores.directory.base import BaseWritableSingleDirectoryStore
+
+
+class LocalDirectoryStore(BaseWritableSingleDirectoryStore):
+
+    local_path: Path
```

Comment:
I'd probably remove this, its nice for auto complete but it might be a string, or will be sometimes and at some points.